x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/signals/open_close_signals_route.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Hapi from 'hapi';
+import { DETECTION_ENGINE_SIGNALS_STATUS_URL } from '../../../../../common/constants';
+import { SignalsRequest } from '../../alerts/types';
+import { setSignalsStatusSchema } from '../schemas';
+import { ServerFacade } from '../../../../types';
+import { transformError, getIndex } from '../utils';
+
+export const setSignalsStatusRouteDef = (server: ServerFacade): Hapi.ServerRoute => {
+  return {
+    method: 'POST',
+    path: DETECTION_ENGINE_SIGNALS_STATUS_URL,
+    options: {
+      tags: ['access:siem'],
+      validate: {
+        options: {
+          abortEarly: false,
+        },
+        payload: setSignalsStatusSchema,
+      },
+    },
+    async handler(request: SignalsRequest, headers) {
+      const { signal_ids: signalIds, query, status } = request.payload;
+      const index = getIndex(request, server);
+      const { callWithRequest } = request.server.plugins.elasticsearch.getCluster('data');
+      let queryObject;
```

Comment:
This is initially `any`
<img width="236" alt="Screen Shot 2019-12-06 at 2 42 01 PM" src="https://user-images.githubusercontent.com/1151048/70358591-f7a04d80-1836-11ea-8ef2-e15223107d7d.png">


You can do this to make it more explicit:

```ts
let queryObject: object | undefined;
```

Also lets the reader know that if `signalsId` or `query` is not defined then `undefined` will end up being the value.